src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs

```diff
@@ -497,30 +498,38 @@ private bool IsPending(KycStatusLogRecord row)
                 partnersDict[p.PublicId] = p.Name;
             }
 
+            Dictionary<string, ClientAccountInformationModel> clients = null;
+            if (items != null)
```

Comment:
Add this check right after Items is retrieveed with GetRecordsForPeriodAsync call.
If it's null then I assume from the code below - we can return null right away, right?

Reply:
refactored